### ISSUE TITLE
[MU4 Issue] Minimum measure width should be greater by default in new scores

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#10395]
https://user-images.githubusercontent.com/90187801/151408423-a2f7973a-7c1d-41f5-b07d-6b01a1a06759.mp4

TO DO..

**Describe the bug**
Minimum measure width should be greater by default in new scores created from (New Score dialog)

Probably, it should depend on instruments amount in score/staff size

I'm thinking  now: 
- for **small scores** it should be about **15sp**
- for **middle/big scores** it should be about **10sp**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by oktophonie** [comment]
10sp would be a reasonable new default. However, changing this should wait until the fix for #10490 (i.e. PR #10511) is completed.

**Comment by MarcSabatella** [comment]
I recall having a discussion about this before, but am not sure where.  My concern remains, if we make the default in new scores larger so they look better immediately upon score creation, that means the occasional empty measure will also be larger in complete scores.  And I'm not sure we really want that.  Earlier we discussed ways around this - ways to make completely empty systems spread out while still allowing for more normal-sized empty measures when they occur within a score.  Unofrtunately, I've forgotten what solutions we came up with.  One might have been, separate minimums for empty measures in the middle of a score versus "trailing" empty measures.
